.gitignore

```diff
@@ -68,20 +68,11 @@ docs/_build/
 
 # pytest files
 src/radical/entk/tests/__pycache__
-tests/__pycache__
-tests/test_api/__pycache__
-tests/test_functionality/__pycache__
-tests/test_integration/__pycache__
-tests/test_issues/__pycache__
-
+tests/*/__pycache__
```

Comment:
Should we use `**/__pycache__` - we also don't want that in other places...?

Reply:
This was a duplication, but one that is there has a larger scope